example/translations/translations.pro

```diff
@@ -0,0 +1,55 @@
+TEMPLATE = aux
+
+# NOTE: the translations here assume that generic sharing UI system will pick up the
+# translations from specific /usr/share/translations/nemotransferengine/ with
+# specific name where '-' separates locale code at the end.
+# It might work for time being, but it's preferred if plugins take the responsibility
+# for loading translations in sharing plugin if that uses translations, and/or from
+# qml code. Latter commonly by importing an own module where c++ side instantiates
+# and installs QTranslator(s) to the qApp on initializeEngine().
```

Comment:
Actually think by now the plugins are themselves responsible for loading their translation files.